packages/runtime/src/components/utils/types.ts

```diff
@@ -4,4 +4,4 @@ export type SwatchValue = {
   lightness: number
 }
 
-export type ColorValue = { swatch?: SwatchValue; alpha: number }
+export type ColorValue = { swatch?: SwatchValue; alpha?: number }
```

Comment:
The type for this when you `useSwatch` is `number | null`. We avoided a type error by casting in the past. Since I am rewriting I thought it would be good to just update the type. Especially since it is low impact.

Reply:
This type is used all over the place, not just with rich text. Is making this optional not causing any errors in other places?